WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -400,49 +402,43 @@ - (UIStatusBarStyle)preferredStatusBarStyle
     return UIStatusBarStyleLightContent;
 }
 
+- (void)configureStackView
+{
+    UIStackView *stackView = [[UIStackView alloc] init];
+    stackView.translatesAutoresizingMaskIntoConstraints = NO;
+    stackView.axis = UILayoutConstraintAxisVertical;
+    stackView.spacing = 0;
+    [self.view addSubview:stackView];
+
+    [NSLayoutConstraint activateConstraints:@[
```

Comment:
You can use our convenience method: `[self.view pinSubviewtoAllEdgeMargins:stackView];`